config/main.py

```diff
@@ -3369,37 +3369,67 @@ def get_acl_bound_ports():
 
     return list(ports)
 
-#
-# 'table' subcommand ('config acl add table ...')
-#
 
-@add.command()
-@click.argument("table_name", metavar="<table_name>")
-@click.argument("table_type", metavar="<table_type>")
-@click.option("-d", "--description")
-@click.option("-p", "--ports")
-@click.option("-s", "--stage", type=click.Choice(["ingress", "egress"]), default="ingress")
-def table(table_name, table_type, description, ports, stage):
+def expand_vlan_ports(port_name):
     """
-    Add ACL table
+    Expands a given VLAN interface into its member ports.
+
+    If the provided interface is a VLAN, then this method will return its member ports.
+
+    If the provided interface is not a VLAN, then this method will return a list with only
+    the provided interface in it.
     """
     config_db = ConfigDBConnector()
     config_db.connect()
 
+    vlan_members = config_db.get_keys("VLAN_MEMBER")
+
+    members = [v[1] for v in vlan_members if port_name == v[0]]
+    return members or [port_name]
```

Comment:
>[port_name] [](start = 22, length = 11)

Seem you will return `[vlan]` if it is empty. Is it expected?

Reply:
Nope, good catch. Fixed!